# Computationally Maintained Datasets

These datasets are computationally maintained but additional papers may be manually added to this set by a curator during the curation process.

When datasets are computationally added to a publication, large scale papers \(more than 100 interactions per experiment\) are excluded.

* [**AFCS**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:afcs%22) - Interactions from the Alliance for Cell Signaling database [![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/AFCS.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/AFCS.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/AFCS.zip)

  This dataset was obtained from the Alliance for Cell Signaling database. The [Alliance of Cellular Signalling (AFCS)](https://www.nature.com/articles/nature01304) consisted of around 20 institutions which were engaged in a collaborative effort to investigate and understand cellular signalling networks. The AfCS used high-throughput methods to detect protein-protein interactions between signaling molecules expressed in B cells and cardiac myocytes. The AfCS arranged a collaboration with Myriad Genetics to perform large-scale yeast two-hybrid screens. IntAct acted as a data repository of protein-protein interaction data generated by the AFCS project.

* [**Apoptosis**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:apoptosis%22) - Interactions involving proteins with a function related to apoptosis [![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/Apoptosis.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/Apoptosis.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/Apoptosis.zip)

  Datasets of apoptosis relevant protein-protein interaction studies are curated by IntAct curators from peer reviewed literature. These datasets are a resource for biologists seeking to understand protein interaction networks and cell death. Small-scale interactions involving proteins annotated with the GO terms "Apoptosis" are included in this set.

* [**Archaea**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:archaea%22) - Interaction dataset based on Archaea proteins [![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/Archaea.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/Archaea.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/Archaea.zip)

  Archaea are phylogenetically very different from Bacteria and Eukarya and show many differences in their biochemistry from other forms of life. This was considered of interest and peer reviewed literature that is curated is scanned for interactions involving proteins from this group.

* [**PDBe**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:pdbe%22) - Data obtained from the Protein Data Bank Europe [![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/MSD.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/MSD.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/MSD.zip)

  The Protein Data Bank in Europe \(PDB3\) is the European project for the collection, management and distribution of data about macromolecular structures, in collaboration with Worldwide Protein Data Bank \([wwPDB](http://www.ebi.ac.uk/pdbe/)\). IntAct has incorporated a subset of the data from this database involving heterodimeric protein interactions.

* [**NDPK**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:ndpk%22) - Interactions involving proteins containing InterPro domain IPR001564, Nucleoside diphosphate kinase, core.[![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/NDPK.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/NDPK.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/NDPK.zip)

  NDPKs, which play a major role in the synthesis of nucleoside triphosphates other than ATP, also possess other enzymatic activities and are required for cell proliferation, differentiation and development.

  Publications based on this dataset: PMID: [19415463](http://europepmc.org/article/MED/19415463)

* [**Synapse**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:synapse%22) - Interactions of proteins with an established role in the presynapse. [![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/Synapse.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/Synapse.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/Synapse.zip)

  This dataset has been created for proteins-protein interactions involving at least one protein with an established link to the synapse. The list of human, rat and mouse gene names used for computationnally maintaining this dataset are available [here](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/Synapse/misc/gene-names.csv). Interactions made by orthologous proteins have been added manually by IntAct curators.

* [**Virus**](http://www.ebi.ac.uk/intact/search?query=annot:%22dataset:virus%22?conversationContext=7) - Publications including interactions involving viral proteins. [![PSI-MI XML 2.5](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXML25.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi25/datasets/Virus.zip) [![PSI-MI XML 3.0](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miXml30.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psi30/datasets/Virus.zip) [![PSI-MI TAB 2.7](https://raw.githubusercontent.com/intact-portal/intact-portal-documentation/master/assets/miTab27.png)](https://ftp.ebi.ac.uk/pub/databases/intact/current/psimitab/datasets/Virus.zip)

  The MINT and HPIDb databases are major contributors to this dataset.

_Disclaimer: Search results for our datasets are displayed on our old website while we implement this search capacity in our new website._
